### PR TITLE
Implement Subscription and wire up for Delegation on account index page

### LINKED ISSUE #1136: Implement Subscription and wire up for Delegation on account index page
Display delegation on account index page for each account

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1136

### CODE REVIEW

**Review by evilpeach — DISMISSED** [review]
LGTM

File: scan/src/subscriptions/DelegationSub.re [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    }
+  }
+  |}
+];
+
+module StakeConfig = [%graphql
+  {|
+  subscription Stake($delegator_address: String!)  {
+    delegations_view(where: {delegator_address: {_eq: $delegator_address}}) @bsRecord  {
+      amount @bsDecoder(fn: "GraphQLParser.bandAmountOpt")
```
Comment (sorawit) [inline scan/src/subscriptions/DelegationSub.re]: numberExn

File: scan/src/subscriptions/DelegationSub.re [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      delegatorAddress: delegator_address @bsDecoder(fn: "GraphQLParser.addressOpt")
```
Comment (sorawit) [inline scan/src/subscriptions/DelegationSub.re]: addressExn

File: scan/src/subscriptions/DelegationSub.re [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      shares @bsDecoder(fn: "float_of_string")
+    }
+  }
+  |}
+];
+
+module StakeConfig = [%graphql
+  {|
+  subscription Stake($delegator_address: String!)  {
+    delegations_view(where: {delegator_address: {_eq: $delegator_address}}) @bsRecord  {
```
Comment (sorawit) [inline scan/src/subscriptions/DelegationSub.re]: offset, limit, orderby

File: scan/src/subscriptions/DelegationSub.re [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    ApolloHooks.useSubscription(
+      MultiConfig.definition,
+      ~variables=
+        MultiConfig.makeVariables(~delegator_address=delegatorAddress |> Address.toBech32, ()),
+    );
+  result |> Sub.map(_, internal => internal##delegations);
+};
+
+let getStakeList = delegatorAddress => {
+  let (result, _) =
```
Comment (sorawit) [inline scan/src/subscriptions/DelegationSub.re:68]: offset, limit

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by prin-r — DISMISSED** [review]

**Review by prin-r — DISMISSED** [review]

**Review by sorawit — DISMISSED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**